library/ReplicatedTable.cs

```diff
@@ -2378,19 +2189,14 @@ private void MergeOperations(TableBatchOperation batch1, TableBatchOperation bat
                 //  1. Store the tombstone entries in a seperate table/blob, whenever an entry is deleted.
                 //  2. Store the tombstone entries in other rows of the existing table in a separate column.
 
-                writeHeadTable.CreateIfNotExists(null, null);
-
-                CloudTableClient readHeadTableClient = txnView[txnView.ReadHeadIndex];
-                CloudTable readHeadTable = readHeadTableClient.GetTableReference(this.TableName);
+                writeHeadTable.CreateIfNotExists();
 
-                TableRequestOptions options = new TableRequestOptions();
-                options.RetryPolicy = new LinearRetry(TimeSpan.FromMilliseconds(300), RETRY_LIMIT);
+                TableServiceClient readHeadTableClient = txnView[txnView.ReadHeadIndex];
+                TableClient readHeadTable = readHeadTableClient.GetTableClient(this.TableName);
 
                 DateTime startTime = DateTime.UtcNow;
-                TableQuery<DynamicReplicatedTableEntity> query =
-                    (from ent in readHeadTable.CreateQuery<DynamicReplicatedTableEntity>()
-                    where ent._rtable_ViewId >= viewIdToRecoverFrom
-                    select ent).WithOptions(options);
+                var query = readHeadTable.Query<DynamicReplicatedTableEntity>(ent =>
+                    ent._rtable_ViewId >= viewIdToRecoverFrom);
```

Comment:
do we not need the request options ?

Reply:
Synced offline to get more context. This originally was added to address issue where network issue would cause rtable operation to hang and never return, so main motivation to add these options is to induce some maximum execution time before throwing an exception.

Confirmed with ConsoleApp tests that default TableServiceClient constructor uses default values for Azure.Core.RetryOptions. 
![image](https://github.com/Azure/rtable/assets/122410443/024cc3f6-4b40-4456-844f-d9d2869844e4)


And created TableClient from GetTableClient method call also picks up these same default values.
![image](https://github.com/Azure/rtable/assets/122410443/8d65747b-218b-4f40-81c6-1b90b2f9698e)

No need to add code changes to enforce 10 retries and a 300ms delay, as the critical aspect of the original change was to make sure call has timeout.